libtransmission/variant-json.cc

```diff
@@ -165,10 +165,50 @@ void action_callback_PUSH(jsonsl_t jsn, jsonsl_action_t /*action*/, struct jsons
         }
     } while (++in != end);
 
-    *setme = val;
+    setme = val;
     return true;
 }
 
+template<typename Iter>
+void decode_single_uchar(char const*& in, char const* const in_end, Iter& buf16_out_it)
+{
+    constexpr unsigned escaped_uchar_length = 6;
```

Comment:
Please follow .clang-tidy rule `{ key: readability-identifier-naming.ConstexprVariableCase,    value: CamelCase  }`

```suggestion
    static auto constexpr EscapedUcharLength = 6U;
```